go.mod

```diff
@@ -41,10 +41,10 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/go-jose/go-jose/v4 v4.0.5 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
-	github.com/klauspost/cpuid/v2 v2.2.9 // indirect
+	github.com/klauspost/cpuid/v2 v2.2.10 // indirect
 	github.com/kr/pretty v0.3.0 // indirect
-	github.com/libdns/libdns v0.2.2 // indirect
-	github.com/miekg/dns v1.1.62 // indirect
+	github.com/libdns/libdns v1.0.0-beta.1 // indirect
```

Comment:
This dependency is probably causing problems. Not sure why it is downgraded so severely.